docs/data/data-grid/row-height/row-height.md

```diff
@@ -78,6 +77,15 @@ Add padding to the cells to increase the space between the content and the cell
 
 :::
 
+### Column virtualization
```

Comment:
I wonder if there is purpose in refering to this in https://deploy-preview-15541--material-ui-x.netlify.app/x/react-data-grid/virtualization/. It might be noise, so might be best not to keep things simple.